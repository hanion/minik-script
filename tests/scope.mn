// scope test
global := 1.1;

{
	print global; // 1.1
	global = 2.2;
	print global; // 2.2
	local := 3;
	print local; // 3
}

print global; // 2.2
print "should give undefined variable error ->";
print local;
