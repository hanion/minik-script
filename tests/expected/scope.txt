1.100000
2.200000
3.000000
2.200000
should give undefined variable error ->
[ERROR] [line 14], Undefined variable 'local'.
